Appraisals

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 case RUBY_VERSION
-when '2.7.7', '3.0.6', '3.1.3', '3.2.1'
+when '2.7.7', '3.0.6', '3.1.3', '3.2.1' then
```

Comment:
Curious about the use of trailing `then`? Is it a rubocop violation?

Reply:
Honestly, I was just copy pasting a few things across multiple Appraisal files and felt it looked nice with the `then` as opposed to without.

Reply:
I don't have any strong feelings, just that it acts like a `;` so it's syntactic sugar

Reply:
👍